frontend/src/__tests__/cypress/cypress/e2e/acceleratorProfiles/AcceleratorProfiles.cy.ts

```diff
@@ -49,17 +53,41 @@ describe('Accelerator Profile', () => {
       .findDescription()
       .contains('Lorem, ipsum dolor sit amet consectetur adipisicing elit. Saepe, quis');
     tableRow.shouldHaveIdentifier('tensor.com/gpu');
+
```

Comment:
can you add `sorting and searching` to the test name description? something like `list accelerator profiles and sorting and searching`?